Library/Formula/aide.rb

```diff
@@ -0,0 +1,27 @@
+class Aide < Formula
+  desc "File and directory integrity checker"
+  homepage "http://aide.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/aide/devel/0.16a2/aide-0.16a2.tar.gz"
+  sha256 "b52451816bc85409ea09dc612e32823336f78438afd28248c252912ea8b91b87"
+  head "http://git.code.sf.net/p/aide/code.git"
+
+  depends_on "automake" => :build
```

Comment:
If these aren't required for building outside of the `head` they should be tucked inside a `head do` block.